Apps/phcheckpointfirewall/checkpoint_connector.py

```diff
@@ -196,11 +197,32 @@ def _logout(self, action_result):
 
         if phantom.is_fail(ret_val):
             self.save_progress("Failed to logout: {}".format(action_result.get_status_message()))
-            return action_result.get_status()
+            return action_result.get_status(), action_result.get_status_message()
 
         self._sid = None
 
-        return phantom.APP_SUCCESS
+        return phantom.APP_SUCCESS, "Successfully logged out of session"
+
+    def _logout_session(self, param):
+
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        if not(self._login(action_result)):
+            return action_result.get_status()
+
+        sid_exisitng_session = param.get('session_id')
```

Comment:
typo: 
```suggestion
        sid_existing_session = param.get('session_id')
```

also since this is a required parameter would be clearer to use `[]` access

Reply:
changes incorporated in e7ce706